### ISSUE TITLE
WCAG profile goes beyond WCAG

### ISSUE THREAD

**Issue by DavidMacDonald** [wet-boew/wet-boew-wpss#16]
WCAG profile goes beyond WCAG, and the quick check is really the WCAG, No? 

As a WCAG team member, I probably would have voted for most of those failures for WCAG, but I don’t think industry would have let us get consensus.
